src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public int getRandomBall() {
+        Random randomNumberBall = new Random();
```

Comment:
```suggestion
        Random random = new Random();
```
and move it outside of method, as class field